manifests/master/storeconfigs.pp

```diff
@@ -3,6 +3,7 @@
 class puppetdb::master::storeconfigs (
   $puppet_conf = $puppetdb::params::puppet_conf,
   $masterless  = $puppetdb::params::masterless,
+  $enable      = false,
```

Comment:
I think this should default to `true` in case anyone applies it outside the scope of of `puppetdb::master::config` otherwise this would break their storeconfigs setting.

Reply:
My rationale was that 'false' is [the puppet default](https://puppet.com/docs/puppet/6.0/configuration.html#storeconfigs) for 'storeconfigs'.   (IMO) it makes sense to be false in `storeconfigs.pp`, which is more like a private class.. which is then overridden by a call from `puppetdb::master::config` which pops it to 'true' in order to preserve module behavior for people using the module already.  Very little surprises, except the one you called out: people who have gone rogue.  I'd say this comes down to "is puppetdb::master::storeconfigs a private class?"

For principle-of-no-surprises I can see doing it your way, and this is not a hill I will choose to die on.  Consider this my one token piece of resistance; if you think yours is the better approach, I'll adjust accordingly.

Reply:
Changed as requested based on no further discussion.